src/web5.js

```diff
@@ -71,18 +71,19 @@ class Web5 extends EventTarget {
       if (dwnNodes) {
         return this.#send(dwnNodes, { author, data, message, target });
       }
-      return { status: { code: 422, detail: 'No DWN endpoints present in DID document. Request cannot be sent.' } };
+      return { status: { code: 400, detail: 'No DWN endpoints present in DID document. Request cannot be sent.' } };
     }
 
-    return { status: { code: 422, detail: 'Target DID could not be resolved' } };
+    return { status: { code: 400, detail: 'Target DID could not be resolved' } };
   }
 
-  async #createSignedMessage(resolvedAuthor, message, data) {
-    const authorizationSignatureInput = this.#dwn.SDK.Jws.createSignatureInput({
-      keyId: resolvedAuthor.did + '#key-1',
-      keyPair: resolvedAuthor.keys,
+  async #createSignedMessage(author, resolvedAuthor, message, data) {
+    const keyId = '#dwn';
```

Comment:
this seems like a really odd choice of key name.  is the idea to try to hide this from callers?  if so, can we use a `Symbol` instead?

Reply:
FWIW, the key name was statically defined as `key-1` before, so I'd argue `#dwn` is "less odd" than `key-1` 😄 

Two things to keep in mind:
1. We haven't introduced a key store to Web5 JS and or at least made it possible for a developer to provide their own.  The key store work has a number of open questions we need to work through, and that will require refactoring all of the methods that touch keys.  We could opt to address then.
2. Since the key store work is still pending, we have been relying on the `keyId` specified when the ION DID is created to match the `keyId` in the `createSignedMessage()` method.  If there's a mismatch, any calls that require message signing fail.  To really make an improvement here what we ought to do is select from one of:
    - Make the `keyId` used for signing messages configurable on a per DID basis WITHOUT requiring it to be passed in on every call to a `web5.dwn.records.*`, `web5.dwn.protocols.*`, etc. call.
    - Make the `keyId` a "well known" that could be a Symbol which is used consistently in both places.  This would mean that an existing DID could not be imported unless it conforms to this approach.  There may be some consideration for doing this even though its restrictive based on how encryption works in `dwn-sdk-js`. 

There is some discussion needed here with @csuwildcat

Reply:
Opened Issue #44 to track this and make a decision.

Reply:
my immediate reaction to this was "are you trying to somehow grab a private field from another object?", so I thought it was extremely odd :/

how about we do something ultra verbose like `"web5-message-signing-key"` so it's explicitly clear what it's for (and would allow for other code to explicitly "opt in" to this if they want to customize it for some reason)